tikv/kv.go

```diff
@@ -195,9 +198,9 @@ func WithPool(gp Pool) Option {
 }
 
 // WithPDHTTPClient set the PD HTTP client with the given address and TLS config.
-func WithPDHTTPClient(tlsConf *tls.Config, pdaddrs []string) Option {
+func WithPDHTTPClient(pdAddrs []string, tlsConf *tls.Config) Option {
```

Comment:
Does it make a compatibility error to TiDB?

Reply:
When upgrading client-go, we will naturally perceive this modification before and after compilation. I don't think it will cause compatibility problems.